com.regnosys.rosetta.lib/src/main/java/com/rosetta/model/lib/annotations/RosettaChoiceRule.java

```diff
@@ -4,6 +4,8 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 
+// @Compat. This is not used anymore. All conditions are now `RosettaDataRule`s.
+@Deprecated
 @Documented
 @Retention(RetentionPolicy.RUNTIME)
 public @interface RosettaChoiceRule {
```

Comment:
Cleanup 1: there is only one type of condition; `RosettaDataRule`s. `RosettaChoiceRule`s are deprecated.